### DISCUSSION TITLE
A possibility to disable vanilla recipes

### DISCUSSION THREAD

**Discussion by Razorxd1993** [Zrips/CMI#8174] [Feature Requests]
It would be cool to have a way to disable individual recipes, like with the CustomCrafting plugin.

Kind regards

**Comment by CloudeLecaw** [comment]
You can disable vanilla crafts allready. hold the craftingresult in hand and use /recipe and click on the greem wool.
The wool become red and the recipe is disabled. (maybe needs restart)
same can be done with custom crafts if they implent their recipe in the same way as vanila does.
If not then you can create a custom shaped recipe that craft some nonsense like..a stick.. dirtblock..what ever.

**Comment by Razorxd1993** [comment]
Hi thanks for your feedback
Unfortunately this doesn't work as it should, if you disable the crafting recipes (Green Wool, not Grey) then they only stay disabled until the next server restart and after a restart they are active again.

**Comment by CloudeLecaw** [comment]
thats not true. if they got removed on restart than there is a problem with your recipes file.
i could upload you our file, start the server with it an you would see if stays.  maybe zrips can do some improvemts here because its not tje first time i heard others saying their disabled recipes got removed after restart ...

there ia our Recipes.yml. start a server with it you will see on log that 6 recipes are disabled.

https://pastebin.com/0g5HBnit
